src/lib/Commands.tsx

```diff
@@ -179,7 +179,7 @@ export function CommandsPalette() {
                 </CommandItem>
                 <CommandItem href="">GitHub</CommandItem>
                 <CommandItem href="">Contact</CommandItem>
-                <CommandItem href="">RSS</CommandItem>
+                <CommandItem href="rss.xml">RSS</CommandItem>
```

Comment:
Should we add a leading slash here?

```suggestion
                <CommandItem href="/rss.xml">RSS</CommandItem>
```